project.clj

```diff
@@ -13,7 +13,8 @@
                  [secretary "1.2.3"]]
 
   :plugins [[lein-environ "1.0.2"]
-            [lein-asset-minifier "0.2.8"]]
+            [lein-asset-minifier "0.2.8"]
+            [lein-cljfmt "0.3.0"]]
```

Comment:
Maybe, idk - I think it's best to keep it up-to-date. Could you actually run cljfmt in this PR so it passes? Then I'm happy to merge it